docs/src/bootstrap.md

```diff
@@ -275,6 +275,38 @@ spec:
     as queries are run as a superuser and can disrupt the entire cluster.
     An error in any of those queries interrupts the bootstrap phase, leaving the cluster incomplete.
 
+Moreover you can specify a list of secrets and/or configmaps which contains sql files that will be executed after the database is created and configured. These sql files will be executed as the *superuser* (`postgres`), connected to the database specified in the initdb:
```

Comment:
the proper words should be `Secrets` and  `ConfigMaps`

Reply:
done